client/src/api/addEvent.js

```diff
@@ -0,0 +1,10 @@
+export function setEvent(title, date, descr, background, user) {
+  const datas = { title, date, descr, background, owner: user };
+  return fetch("/api/addEvent", {
```

Comment:
This is a very uncommon name for a route.
Routes are document orientated. See json-server implementation or RESTful guidelines
I would suggest to call it `/api/events` for both, GET and POST requests.